app/install/kubernetes.md

```diff
@@ -1,3 +1,3 @@
 ---
-redirect_to: /2.0.x/kong-for-kubernetes/install
+redirect_to: /2.1.x/kong-for-kubernetes/install
```

Comment:
could we replace `2.1.x` with `latest`? I think `latest` will always redirect to the latest version, then we don't need to keep updating these